### ISSUE TITLE
Empty description set for some frameworks

### ISSUE THREAD

**Issue by siuc-nate** [CredentialEngine/CredentialRegistry#624]
These frameworks seem to return empty description sets:
```
ce-65645d93-ec5b-4757-a56a-89797c8fecaa
ce-8101de56-2081-444f-adce-61ad76bf3ac9
ce-82064e39-722c-4c11-abfc-c1081654d012
ce-c4a5fad2-e2c1-42ff-b43a-cb4ee6ee4a58
ce-f248712a-8c37-4e1f-a51d-387a869f93c4
ce-b7843020-0875-4a95-b959-6f348687c5c3
ce-3ded4bf3-ce7e-4831-9e6f-f0bb7d73a56f
ce-da3e03fb-69a8-42fd-a961-1f98c538c069
ce-9fa74630-e74b-451b-85f0-95868abcbaa5
ce-633de50b-6322-40cb-a0e2-0300da65e0c1
```
There are likely others, but these are the ones I've spotted.
![image](https://user-images.githubusercontent.com/21346519/225414662-b50eb1a5-f62d-46ad-b3ef-1b19ca84d8ff.png)

**Comment by siuc-nate** [comment]
The following frameworks return description sets and related resources, but no actual data:
![image](https://user-images.githubusercontent.com/21346519/228020694-97a93308-4502-4d03-a292-ec83a8d6bc7a.png)

```
[ "ce-aa1a8d4f-d347-40ed-9936-5579aaf6ee4d", "ce-712820bb-b6ed-485e-9657-f47d6407dbc5", "ce-1a0b5c43-f3c5-45af-83f6-698da60b9fd8", "ce-cf8c0f27-230c-4ed9-8d99-30f6bdec4252", "ce-ece7cefa-3c0f-42fe-9e51-f24e5489aa0c", "ce-b3485f5b-f90a-422c-99f7-8901cffa5556", "ce-d188059a-dfeb-4822-833d-9128237fd0a9", "ce-474571e3-36e4-4994-8b2e-29f08e5e3f77", "ce-2d0d1829-81ca-437f-b67b-73be1fdcd86e", "ce-f204c85c-1a3e-4ac7-8a3c-705e5833ac9d", "ce-81676af1-c455-49b7-969c-bd1a573d0e67", "ce-729850fe-547e-4bc7-a8de-2782b053b0bb", "ce-463e6042-1b6b-4b53-8c16-ec3ac1605d06", "ce-64d8493d-de8d-41db-81e3-4b60e57506c4", "ce-02a8adbd-3da9-487c-8f11-3d97cfd7aedb", "ce-2d6925c7-e3bc-4ffb-a35f-bab30d8764d1", "ce-4ef79a9f-ffe5-4f99-8fb3-fc716414586e", "ce-da3fd37b-b835-4989-8f1c-c0880c192fc8", "ce-c277c93c-81ae-4e7c-b930-4c30ce3dc71b", "ce-a8ad573d-ed45-4d57-9ffb-43f4ea623e9d", "ce-226f9c67-8497-4f03-8bd0-6d7b1c072bbc", "ce-27115b6c-bbbc-4ff2-a6e5-49b8e1f022e9", "ce-726e57e7-737a-4641-a0d9-018cbc81ed24", "ce-820fa9bf-1f1a-412e-b10e-c07d7c570f60", "ce-5c260fb2-541d-4dce-af4e-d9b2b07633ec", "ce-b4f7e4d1-bc57-4bd3-86f7-4158ed4a997f", "ce-21155df0-ccc4-404d-ad84-00a6559161d6", "ce-77918080-d071-4215-a63c-e507d7d93325", "ce-342b863a-9530-4acc-a980-52a2d8a48cdd", "ce-3c12f3d0-3b10-4c88-ab84-95916605bf57", "ce-6a36a56f-6f46-4467-b523-e897b74468e2", "ce-d734ca0a-6827-43ea-a22e-a0d3c38816ae", "ce-ae88ce1f-7642-47b3-93d3-dfb95b4c4764", "ce-d82041d6-0165-4116-9463-18be163628a1", "ce-56d82978-6658-4e6e-b104-af96fdb30312", "ce-8c808318-86a9-419c-935d-49b9e5e27a11",  "ce-cfe260dd-da86-428e-b68e-c4543936c236", "ce-c623cabc-7a07-4ddb-9cc0-5cb3365823bd", "ce-8b81a0ca-3ae0-4a0a-a50b-cfff95b6294c", "ce-d801ad69-f7fd-4697-b120-2a83c6cbe08a", "ce-c79dac02-ff1f-4932-99f4-7663e5a9d943", "ce-225eca26-805a-46c1-be0d-67ac0b718ffb", "ce-defdb770-3687-4163-a956-17bf91c8a8f2", "ce-85a65088-a565-4db4-be78-c5a738da940d", "ce-3f5d05f1-2e09-4c56-b2b3-8389aa4b0e57", "ce-7f6cf7d8-f1dd-429a-ac76-53003f2b874c", "ce-514c712f-2f96-48fb-91d3-b0e513d6d943", "ce-091cb0be-f68a-4a54-b757-8f28a695504e", "ce-44409cba-15b2-4f25-859f-c1de8d5ac691", "ce-f68e5094-2b57-44aa-b64a-dde583fba87b", "ce-f3511a6c-2481-43b2-bbc8-8bdabc424001", "ce-409cef86-b2b4-474d-9d22-e986533ed170", "ce-10af14fc-2f39-43b6-a11f-916b8430c154", "ce-b399e77d-60e9-4d81-a4c8-872f3f6b6ed5", "ce-a3662b53-1b93-4bfb-b0a3-e8a715cca336", "ce-555ce6a8-945e-45ec-99a5-097111a9fdf4", "ce-19528df5-b086-4386-86be-824764e13d6b", "ce-a8bbd416-9e03-4fb3-85a0-310347b2462b" ]
```

**Comment by excelsior** [comment]
@siuc-nate None of the 58 CTIDs you shared are present in the database. Nine of them are soft-deleted, the rest are absent completetly. Some of those CTIDs had description sets presents, though, which was not supposed to happen. I removed all of such description sets. IOW, the lack of data is not a bug.

I made the `description_set_resources` property always exposed in the response even if it's empty when subresources are requested.
